source/_components/switch.ecoplug.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: EcoPlug/Wion Switch
+description: "Instructions on how to set up an EcoPlug-Compatible switch within Home Assistant."
+date: 2018-06-25 14:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Switch
+ha_release: 0.71
```

Comment:
`0.71` has been released already, maybe set this to the next major release, e.g., currently, `0.73`

Reply:
Noted. Thank you! I'll make all of these changes after clearing the checklist for the device itself.